QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static readonly IList<IMessageListener> _listeners = new List<IMessageListener>();
+
+        public void Start(IConsole messageConsole) {
+            _lineInput = messageConsole.ReadLine();
```

Comment:
Could `_lineInput` be a variable local to this method? (vs being a member variable)